compiler+runtime/src/cpp/clojure/core_native.cpp

```diff
@@ -276,6 +276,19 @@ namespace clojure::core_native
   {
     return __rt_ctx->eval(expr);
   }
+
+// TODO: implement opts
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wunused-parameter"
+  static object_ptr read_string(object_ptr const opts, object_ptr const str)
+#pragma clang diagnostic pop
+  {
+    return __rt_ctx->read_string(runtime::to_string(str));
+  }
+
+  static object_ptr jank_version() {
+    return make_box(JANK_VERSION);
+  }
```

Comment:
Please make sure you run clang-format aon all of these files. You can use `bin/format` if you don't have it hooked up in your editor.

Reply:
Sorry, my bad. Will do 👍